src/popup.html

```diff
@@ -101,7 +100,7 @@ <h3>Current Tab</h3>
       <div id="current-page"></div>
       <label for="container-page-assigned">
```

Comment:
If you add `id="container-page-assigned-label"` attribute here (and don't need to add `id="default-text"` below, you can hide the whole block of elements with the single line mentioned above.